apache/incubator-nuttx
# Ethernet over USB docs and helper scripts

### Summary

* Adds defconfig for Ethernet over USB 2.0 High Speed that works reliably
* Removes other redundant Ethernet over USB defconfigs
* Improves Ethernet over USB helper script and moves it to tools
* Updates SAMA5D3-Xplained README.txt to cover setup and usage

### Impact

* None if you don't use the CDC ECM driver
* CDC ECM driver can now do 4.4 MBytes/s (34 Mbits/s) on the SAMA5D3-Xplained

### Limitations / TODO

* Reliability seems to depend on correct IOB configuration. This one was arrived at via trial and error.

### Testing

* Via an improved tcpblaster, this will be submitted in another PR

### How To Verify 

* Set up via the README.txt instructions and use tcpblaster to verify speed and ability to run for long times.
